packages/edge-functions/dev/node/main.ts

```diff
@@ -159,10 +164,21 @@ export class EdgeFunctionsHandler {
       }),
       {},
     )
-    const { denoPort, success } = await this.initialization
 
-    // TODO: Log error
+    const initMessage = setTimeout(() => {
+      if (this.initialized) {
+        return
+      }
+
+      this.logger.log(
+        'Setting up the Netlify Edge Functions environment. This may take up to a couple of minutes, depending on your internet connection.',
+      )
+    }, 1_500)
+
+    const { denoPort, success } = await this.initialization
     if (!success) {
+      clearTimeout(initMessage)
```

Comment:
nit:
```suggestion
    const initMessageTimeout = setTimeout(() => {
      if (this.initialized) {
        return
      }

      this.logger.log(
        'Setting up the Netlify Edge Functions environment. This may take up to a couple of minutes, depending on your internet connection.',
      )
    }, 1_500)

    const { denoPort, success } = await this.initialization
    if (!success) {
      clearTimeout(initMessageTimeout)
```